doc/users/verify_quote.rst

```diff
@@ -1,33 +1,34 @@
 Verifying Quote
 ===============
 
-A client can verify the SGX quote of the CCF node that it connects to.
-
-First, the client should connect to the node to verify, specifying the ``/node/quote`` endpoint:
+A client can verify the validity of the SGX quote of the CCF node that it connects to. This can be done using the ``verify_quote.sh`` script installed as part of the CCF install or ccf Python package (see :doc:`/quickstart/install`).
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem
-    {"quotes": [{"mrenclave":"<measurement_hash>, "node_id":<node_id>, "raw":"<hex_encoded_raw_quote>"}]}
+    $ verify_quote.sh https://<ccf-node-address> [--mrenclave <mrenclave_hex>] [CURL_OPTIONS]
+    Retrieved 1 accepted code versions from CCF service.
+    Node quote successfully retrieved. Verifying quote...
+    Quote verification successful.
+
+The script verifies the target CCF node's SGX quote by:
+
+1. Retrieving the node's SGX quote via the ``/node/quote`` endpoint.
+2. Verifying that the cryptographic hash of the node's identity public key (as presented in the node's certificate in the TLS session) matches the quote's report data.
+3. Verifying that the quote's measurement (``MRENCLAVE``) matches one of the trusted enclave measurements.
 
-The raw quote should be decoded and output to file for verification via the Open Enclave ``oeverify`` command-line utility:
+A specific trusted SGX enclave measurement can be specified ``--mrenclave <mrenclave_hex>``.
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem | jq .raw | xxd -r -p > ccf_node_quote.bin
+    $ verify_quote.sh https://<ccf-node-address> --mrenclave 3175971c02d00c1a8f9dd23ca89e64955c5caa94e24f4a3a0579dcfb2e6aebf9 [CURL_OPTIONS]
+    Node quote successfully retrieved. Verifying quote...
+    Quote verification successful.
 
-    $ /opt/openenclave/bin/oeverify -r ccf_node_quote.bin -f LEGACY_REPORT_REMOTE
-    Verifying evidence ccf_node_quote.bin...
-    Claims:
-    Enclave unique_id: <ccf_node_mrenclave>
-    Enclave signer_id: <ccf_node_mrsigner>
-    Enclave product_id: <ccf_node_product_id>
-    Enclave sgx_report_data: <ccf_node_report_data>
-    Evidence verification succeeded (0)
+If no enclave measurement is specified, the ``verify_quote.sh`` script automatically verifies it against the code versions currently trusted by the CCF service, as returned by the ``/node/code`` endpoint.
```

Comment:
```suggestion
If no code measurement is specified, the ``verify_quote.sh`` script automatically verifies it against the code versions currently trusted by the CCF service, as returned by the ``/node/code`` endpoint.
```

Reply:
I believe this `mrenclave` is usually referred to as "enclave measurement" rather than "code measurement" (e.g. [here](https://01.org/sites/default/files/documentation/intel_sgx_developer_guide_pdf.pdf))?

Reply:
Fair enough, it's just we talk a lot about the code and not much about initial state here, but ok.